.github/workflows/publish-to-edge.yaml

```diff
@@ -0,0 +1,33 @@
+name: Publish to edge
+
+on:
+  push:
+    branches:
+      - master
+      - main
+      - track/**
+
+jobs:
+  tests:
+    name: Run the tests
+    uses: ./.github/workflows/ci.yaml
```

Comment:
I think to use the `ci.yaml` as a reusable workflow, `ci.yaml` must include an `on: workflow_dispatch` trigger event.  

If you haven't already, I suggest making a separate repo for testing how to use the workflow dispatch, etc., that way you can fine tune.  Unless testing by pushing to this PR is enough, if so then ignore :)

Reply:
That's right, I missed that point in `ci.yaml`. I think it should be `on: workflow_call` though - at least that seems to be working.
I made a [dummy repo](https://github.com/natalian98/template-operator/actions) to trigger the actions on pr and merge. The actions are obviously failing due to no secrets stored, but we can at least see that the tests workflow from `ci.yaml` is re-used and required by `publish-to-edge` as expected.

Reply:
> workflow_call is used to indicate that a workflow can be called by another workflow.

> To manually trigger a workflow, use the workflow_dispatch event.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows